example_test.go

```diff
@@ -311,3 +341,26 @@ func Example_copyArtifactManifestRemoteToLocal() {
 	// Output:
 	// true
 }
+
+// Example_extendedCopyArtifactAndReferrersRemoteToLocal gives an example of
+// copying an artifact along with its referrers from a remote repository to local.
+func Example_Example_extendedCopyArtifactAndReferrersRemoteToLocal() {
+	src, err := remote.NewRepository(fmt.Sprintf("%s/source", remoteHost))
+	if err != nil {
+		panic(err)
+	}
+	dst := memory.New()
+	ctx := context.Background()
+
+	tagName := "latest"
+	// ExtendedCopy will copy the artifact tagged by "latest" along with all of its
+	// referrers from src to dst.
+	desc, err := oras.ExtendedCopy(ctx, src, tagName, dst, tagName, oras.DefaultExtendedCopyOptions)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println(desc.Digest)
```

Comment:
Shouldn't we also check the existence of `exampleSignatureManifestDescriptorArtifactspec` in the local target?

Reply:
How to check? We didn't show this descriptor in the example.